src/SymbolServer.jl

```diff
@@ -42,7 +53,8 @@ function getstore(ssi::SymbolServerInstance, environment_path::AbstractString, p
                     if manifest !== nothing
                         @debug "Downloading cache files for manifest at $(manifest_filename)."
                         to_download = collect(validate_disc_store(ssi.store_path, manifest))
-                        batches = Iterators.partition(to_download, max(1, floor(Int, length(to_download)÷50)))
+                        remove_non_general_pkgs!(to_download)
+                        batches = Iterators.partition(to_download, 10) # download up to 10 at a time
```

Comment:
This definitely seems better than the previous version, but I'm actually starting to wonder why we batch at all? Maybe we should just fire all requests off at the same time async? My guess is that at most this is something like a few hundred HTTP requests, which doesn't strike me as a lot?

Reply:
Ok, updated to no batching. I guess it's worth trying on insider